beacon_node/execution_layer/src/lib.rs

```diff
@@ -1989,6 +2163,38 @@ fn timestamp_now() -> u64 {
         .as_secs()
 }
 
+fn static_valid_tx<T: EthSpec>() -> Result<Transaction<T::MaxBytesPerTransaction>, String> {
```

Comment:
would it be ok if we move this function to `test_utils` since it's only used there and is exclusively for testing?

Reply:
of course, moved here: 0e75d06c5c55cf8da983a733b4656c2761155c31